client/src/components/AttendeeInput.tsx

```diff
@@ -108,11 +116,14 @@ export default function AttendeeInput({ id, onChange, value, type }: AttendeeInp
           multiple
           options={options}
           value={value || []}
-          getOptionLabel={(option) => option}
+          getOptionLabel={(option) => (typeof option === 'object' && option.email ? option.email : '')}
+          noOptionsText=""
           freeSolo
           inputValue={textInput}
           fullWidth
-          onChange={handleSelectionChange}
+          onChange={(_, newValue) => {
+            handleSelectionChange(_, filteredEmails(newValue));
```

Comment:
the job of the method filteredEmails can be done inside the `handleSelectionChange` method as it's only 2 lines of code. And here, we can just do `onChange={handleSelectionChange}`